src/main/mondrian/olap/MondrianProperties.xml

```diff
@@ -1052,6 +1052,18 @@ setting it too high can cause a big delay between the query being marked as canc
         <Default>10000</Default>
     </PropertyDefinition>
     <PropertyDefinition>
+        <Name>CheckCancelOrTimeoutInterval</Name>
```

Comment:
Sorry I just saw this, but we already have another property, CancelPhaseInterval, which does the same. Please standardize on one of the two and make sure to replace all usages of the old property with your new code to handle cancelations as well.